etc/tools/.rat-excludes

```diff
@@ -0,0 +1,33 @@
+target
+MANIFEST.in
+.gitignore
+.gitattributes
+.project
+.classpath
+.rat-excludes
+.*ipynb
+.ipynb_checkpoints/
+.*md
+.*avsc
+.*txt
+.*json
+.*data
+.*log
+.*out
+.*parquet
+.*csv
+.*tsv
+.*iml
+service.properties
+db.lck
+DESCRIPTION
+NAMESPACE
+package-sparkR.sh
+\.lintr
+.*ivy.xml
+\.example-image
+\.ensime
+\.example-image
+\.jvmopts
+\.pairs
+\.travis.yml
```

Comment:
This list was based on Spark's [rat-excludes](https://github.com/apache/spark/blob/master/dev/.rat-excludes). I did not test the data files, so those are eligible to be removed. However, most of the others appear if they are not included. If they are eligible for a header we can add the headers and remove them from the list. However, there are some files which get generated (like *.iml and db.lck) and cannot be removed from this list.